include/net/net_mgmt.h

```diff
@@ -212,6 +234,7 @@ void net_mgmt_event_notify(u32_t mgmt_event, struct net_if *iface);
 int net_mgmt_event_wait(u32_t mgmt_event_mask,
 			u32_t *raised_event,
 			struct net_if **iface,
+			const void **info,
```

Comment:
Don't we need the length too here?

Reply:
No, we will document the net events so that event E meant info of this type T (and thus the implied length). It's not going to happen that 1 event can send a different type of info